demisto_sdk/commands/lint/commands_builder.py

```diff
@@ -55,10 +33,18 @@ def build_flake8_command(files: List[Path], py_num: str) -> str:
         str: flake8 command
     """
 
-    command = f"{get_python_exec(py_num)} -m flake8"
+    command = "flake8 "
+
+    # This is the same config used in `tox.ini` file in content
+    # We will probably want to use that in the future for this and all linters
+    command += '--ignore=W605,F403,F405,W503 '
```

Comment:
Each param should take care of its own spacing.
```suggestion
    command += ' --ignore=W605,F403,F405,W503'
```